src/pages/main.tsx

```diff
@@ -0,0 +1,45 @@
+import { useState } from "react";
+import styled from "styled-components";
+import { getBooks } from "../api/books";
+
+import Header from "../component/header";
+import Product from "../component/product";
+
+const Block = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  margin-left: 6%;
+  margin-right: 6%;
+  margin-top: 6%;
+`;
+
+function MainPage() {
+  const [books, setBooks] = useState([]);
+
+  const showBooks = () => {
+    return books.map((item) => {
+      return (
+        <div>
+          <Product key={item["id"]} id={item["id"]} />
```

Comment:
why are you accessing properties on the item object like this? the `.` notation is much simpler => item.id